test-integration/test_integration/fixtures/apt-packages/cog.yaml

```diff
@@ -0,0 +1,10 @@
+build:
+  gpu: true
+  python_version: "3.9"
+  python_packages:
+    - "torch==2.0.1+cu118"
+  system_packages:
+    - "git"
+  run:
+    - command: git --version
```

Comment:
What's the relationship between torch and git here? Is the torch package needed for the test?

Reply:
No relationship, I will remove

Reply:
Resolved in [96f51c6](https://github.com/replicate/cog/pull/2104/commits/96f51c6108424122ddb4ee7e89bb209a9c5c3164)